config/initializers/money.rb

```diff
@@ -82,7 +82,7 @@
 
   # If you would like to use I18n localization (formatting depends on the
   # locale):
-  # config.locale_backend = :i18n
+  config.locale_backend = :i18n
```

Comment:
This change is not required for this PR.

Reply:
done.